rest-assured-tests/src/test/java/commonTestClasses/FailedResponse.java

```diff
@@ -2,23 +2,12 @@
 
 import com.google.gson.Gson;
 import io.restassured.response.Response;
+import lombok.Getter;
 
 public class FailedResponse {
-    private int status;
-    private String error;
-    private String message;
-
-    public int getStatus() {
-        return status;
-    }
-
-    public String getError() {
-        return error;
-    }
-
-    public String getMessage() {
-        return message;
-    }
+    private @Getter int status;
+    private @Getter String error;
+    private @Getter String message;
```

Comment:
you can set @Getter annotation on class
(by the way, you can always check out code in service itself to get examples)

Reply:
ok, tnx. I've updated it.